index.html

```diff
@@ -3,34 +3,37 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Fredoka+One" rel="stylesheet">
   </head>
   <body>
-    <div>
+    <div class="container">
```

Comment:
Nice, whole page in a container -- maybe a good place for a huuuuge grid to help with layout (with which you don't need more help, as your layout looks pixel-perfect). Do you style this class anywhere?